### ISSUE TITLE
Metadonnées - Ajouter l'adresse mail après le nom+prénom pour la lisibilité

### ISSUE THREAD

**Issue by pierre56** [PnX-SI/GeoNature#2691]
# Metadonnées - Ajouter l'adresse mail après le nom+prénom pour la lisibilité
 
## Contexte
Ayant beaucoup d'utilisateurs (20k), et ayant parfois 3 comptes pour la même personne, en fonction de son statut ou outil lors de la saisie. 
(Le compte personnel + le compte pro + compte obsolète)
Une amélioration simple mais pratique serait d'afficher l'adresse mail entre parenthèse après le prenom/nom lors de la sélection des acteurs.
Cet affichage de l'adresse mail ne serait disponible que pour les utilisateurs ayant au moins les droits de référent.

## Exemple
Entre ces 2 comptes, je dois vérifier si j'ai le bon id en bdd après sélection : 
Jean Trucmuche ( Jean.Trucmuche@bretagne-vivante.org)
Jean Trucmuche ( J.Tmche@gmail.com)

-------
Question bonus : 
est ce normal/cohérent si lorsque je renseigne un organisme + une personne, 
j'ai la possibilité d'ajouter une personne n'étant pas membre de l'organisme ?

il ne faudrait pas un filtre supplémentaire pour afficher seulement les membres de l'organisme ?
Cela me permettrait de ne pas se poser la question du dessus.

Merci :D
Pierre

**Comment by TheoLechemia** [comment]
Bonjour,
Je pense que si on le fait, il faut l'élargir à tous les formulaires ou on renseigne des personnes / observateurs. ça pourrait être controlé par un paramètre, et par défaut désactivé.
Pour la seconde question, tel que c'est pensé, c'est qu'il peut y avoir un organisme ET une personne (pas forcément de cet organisme là). Ex sur un JDD on a l'organisme X qui est financeur et la personne Y (d'un tout autre organisme) qui est producteur.

**Comment by camillemonchicourt** [comment]
Oui, je comprends le contexte, mais ça peut être assez lourd au niveau visuel sur les formulaires, voire non souhaitable de diffuser les emails, pour un cas assez peu fréquent.
Peut-être qu'il serait moins gênant de pouvoir afficher l'organisme de l'utilisateur entre parenthèse après son nom, plutôt que son email, notamment avec les questions de RGPD.

Par ailleurs, je ne vois pas à quoi tu fais référence quand tu dis "ayant au moins les droits de référent". Il n'y a pas de concept de référent dans les permissions de GeoNature.

**Comment by pierre56** [comment]
Merci @TheoLechemia et @camillemonchicourt pour vos réponses.
Vous avez compris les problématiques RGPD et lisibilité de l'interface qui me posent aussi problème.

La solution d'avoir l'organisme entre parenthèse serait suffisante pour répondre au besoin car en fonction du compte, le role appartient a un organisme spécifique.

--------

Pour la notion de référent par contre, j'ai du louper ou pas compris un truc.
Je maj mon install geonature depuis la 2.7.5 donc il peut rester des trucs qui trainent.
Mais le role de référent est bien présent et utilisé :

![image](https://github.com/PnX-SI/GeoNature/assets/26764293/7b9a5fb3-5a92-401c-8d12-c7da2e11d63e)
![image](https://github.com/PnX-SI/GeoNature/assets/26764293/162fda9c-d655-4b55-bf74-92ff07ce09f5)

et pour faciliter l'administration, j'ai créé un groupe grp_referent ayant les profils de " Rédacteur - Référent - Modérateur - Validateur" 
afin que les utilisateurs étant "référents des observatoires régionaux de bretagne" aient des droits élevés afin de saisir des données, des JDD et pouvoir valider les données, ce sont quasiment des admin secondaires.

![image](https://github.com/PnX-SI/GeoNature/assets/26764293/85809595-8495-4925-b01c-cd75170239c8)
-------

Merci 😄

**Comment by camillemonchicourt** [comment]
OK, le fait d'ajouter l'organisme après le nom de l'utilisateur me semble plus souhaitable que leur email. Mais à voir si on fait partout pour tous ou si on en fait un nouveau paramètre, car comme évoque, ça alourdit la lisibilité. Donc surement nécessaire d'en faire un paramètre à `False` par défaut.
Comme dit Théo, si on fait ça, il faudrait le faire partout où on a ce champs. Mais à voir si on le fait aussi dans les endroits où on affiche la valeur de ce champs, pas seulement dans les formulaires...

-----------------------

Concernant les permissions, en effet le profil "Référent" dans UsersHub n'a rien à voir et aucun effet dans GeoNature. 
Les profils sont utilisés dans d'autres applications connectées à UsersHub (TaxHub par exemple) mais pas GeoNature qui gère ses permissions directement dans GeoNature. Voir la doc sur le sujet - https://docs.geonature.fr/admin-manual.html#gestion-des-droits

D'ailleurs chez nous, quand on veut associer un utilisateur ou groupe d'utilisateur à l'application GeoNature, seul le profil "Lecteur" est proposé : 
![image](https://github.com/PnX-SI/GeoNature/assets/4418840/540d3fde-8933-4b5b-a39d-3e81d10fe943)
Donc je ne sais pas pourquoi chez vous, tous les profils sont proposés pour l'application GeoNature.
C'est peut-être un ménage de la BDD qui n'a pas été fait, je ne sais plus.

C'est très bien d'avoir votre groupe des référent, mais il faut juste leur donner un profil "Lecteur" dans GeoNature pour qu'ils y accèdent et ensuite donner des permissions GeoNature à ce groupe dans le module Admin de GeoNature.
